src/components/personal/PersonalPage.vue

```diff
@@ -0,0 +1,194 @@
+<template>
+	<div class="personal-page">
+		<v-avatar :tile="false" :size="130"
+							class="base-main__page__cart__avatar">
+			<img :src="imgPath"/>
+		</v-avatar>
+		<br>
+		<v-btn
+			v-if="loggedUser()"
+			small
+			class="personal-page__button"
+			@click="openEdit()"
+		>
+			<i class="material-icons">
+				settings
+			</i>
+		</v-btn>
+		User: {{user.nick_name}}
+		<div class="personal-page__nickname">
+			<h1>{{user}}</h1>
+		</div>
+		<div v-if="edit" class="personal-page__edit-area">
+			Enter Your Nickname here:
+			<v-text-field
+				v-model="editUser.nick_name"
+				label="Enter Your NickName"
+				type="text"
+			/>
+			if you want to change your password, fill next:
+			<v-text-field
+				v-model="old_password"
+				:rules="passwordRules"
+				label="Enter Old Password"
+				type="text"
+
+			/>
+			<v-text-field
+				v-model="new_password"
+				:rules="passwordRules"
+				label="Enter New Password"
+				type="text"
+			/>
+			<v-text-field
+				v-model="confirm_new_password"
+				:rules="passwordRules"
+				label="Confirm New Password"
+				type="text"
+				required
+			/>
+
+			<v-btn @click="saveEditUser()">Save</v-btn>
+		</div>
+		<br>
+
+	</div>
+</template>
+
+<script>
+	import UserRepository from '../../classes/user/UserRepository.js'
+	import User from '../../classes/user/User'
+
+	export default {
+		name: "PersonalPage",
+		components: {},
+		data() {
+			return {
+				user: {
+					type: User,
+					required: true
+				},
+				editUser: {
+					type: User,
+					required: true
+				},
+				edit: false,
+				new_password: '',
+				old_password: '',
+				confirm_new_password: '',
+				passwordRules: [
+					v => !!v || 'Password is required',
+					v => (v && v.length >= 6) || 'Password must contain 6 characters'
+				],
+			}
+		},
+		inject: ['provideNick'],
+		props: {
+			id: Number
+		},
+		computed: {
+			imgPath() {
+				return require('../../assets/' + localStorage.getItem('id') + '.png')
+			}
+		},
+		created() {
+			this.getUser()
+		},
+		methods: {
+			async getUser() {
+				const id = this.$route.params.user_id;
+				const userRepository = new UserRepository();
+				this.user = await (userRepository.getUserById(id))
+			},
+			async changePassword() {
+				if (this.old_password === this.old_password) {
```

Comment:
if (this.old_password === this.old_password) what?

Reply:
It always true?

Reply:
fixed